src/wp-includes/template.php

```diff
@@ -699,16 +700,20 @@ function get_attachment_template() {
  * @return string The template filename if one is located.
  */
 function locate_template( $template_names, $load = false, $load_once = true, $args = array() ) {
+	$stylesheet_path = get_stylesheet_directory();
+	$template_path   = get_template_directory();
+	$is_child_theme  = is_child_theme();
```

Comment:
```suggestion
	$is_child_theme  = $template_path !== $stylesheet_path;
```

The `is_child_theme()` function calls the `get_template_directory()` and `get_stylesheet_directory()` functions again, so let's utilize the already defined local variables, as Jonny also suggested in https://github.com/WordPress/wordpress-develop/pull/5160#discussion_r1319825399.

Reply:
Updated in https://github.com/WordPress/wordpress-develop/pull/5160/commits/b9eb49db7a8af7bef0931d35100f88ef019ab9ef